thoth/storages/graph/postgres.py

```diff
@@ -2254,13 +2256,16 @@ def get_si_aggregated_python_package_version(
         """
         index_url = GraphDatabase.normalize_python_index_url(index_url)
         with self._session_scope() as session:
-            query = self._construct_query_get_si_aggregated_python_package_version(
+            query = self._construct_query_GET_SI_AGGREGATED_PYTHON_PACKAGE_VERSION(
```

Comment:
Would using the capitalization work? The function name seems to be lowercase - https://github.com/thoth-station/storages/blob/e2ad480759750823cb67be9ba57f2e006169c42f/thoth/storages/graph/postgres.py#L2160

Reply:
Oh, I must have done this accidentally when I capitalized the cache size...